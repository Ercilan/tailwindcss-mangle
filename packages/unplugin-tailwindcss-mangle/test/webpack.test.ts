import webpack from 'webpack'
import { describe, it, expect } from 'vitest'

describe('webpack build', () => {
  // webpack({}, (err, stats) => {
  //   if (err || stats.hasErrors()) {
  //     // ...
  //   }
  //   // Done processing
  // });
})
