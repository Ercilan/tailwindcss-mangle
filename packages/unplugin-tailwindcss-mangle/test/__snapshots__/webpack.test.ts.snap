// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`webpack build > common > assets 1`] = `
{
  "index.js": "(()=>{"use strict";document.querySelector("#app").innerHTML='\\n<main class="flex min-h-screen flex-col items-center justify-between p-24">\\n<div class="z-10 w-full max-w-5xl items-center justify-between font-mono text-sm lg:flex">\\n  <p\\n    class="fixed left-0 top-0 flex w-full justify-center border-b border-gray-300 bg-gradient-to-b from-zinc-200 pb-6 pt-8 backdrop-blur-2xl dark:border-neutral-800 dark:bg-zinc-800/30 dark:from-inherit lg:static lg:w-auto  lg:rounded-xl lg:border lg:bg-gray-200 lg:p-4 lg:dark:bg-zinc-800/30">\\n    Get started by editing&nbsp;\\n    <code class="font-mono font-bold">pages/index.tsx</code>\\n  </p>\\n  <div\\n    class="fixed bottom-0 left-0 flex h-48 w-full items-end justify-center bg-gradient-to-t from-white via-white dark:from-black dark:via-black lg:static lg:h-auto lg:w-auto lg:bg-none">\\n    <a class="pointer-events-none flex place-items-center gap-2 p-8 lg:pointer-events-auto lg:p-0"\\n      href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template-tw&utm_campaign=create-next-app"\\n      target="_blank" rel="noopener noreferrer">\\n      By\\n    </a>\\n  </div>\\n</div>\\n\\n<div\\n  class="relative flex place-items-center before:absolute before:h-[300px] before:w-[480px] before:-translate-x-1/2 before:rounded-full before:bg-gradient-radial before:from-white before:to-transparent before:blur-2xl before:content-[\\'\\'] after:absolute after:-z-20 after:h-[180px] after:w-[240px] after:translate-x-1/3 after:bg-gradient-conic after:from-sky-200 after:via-blue-200 after:blur-2xl after:content-[\\'\\'] before:dark:bg-gradient-to-br before:dark:from-transparent before:dark:to-blue-700/10 after:dark:from-sky-900 after:dark:via-[#0141ff]/40 before:lg:h-[360px]">\\n  <img class="relative dark:drop-shadow-[0_0_0.3rem_#ffffff70] dark:invert" src="/vite.svg" alt="Next.js Logo"\\n    priority />\\n</div>\\n\\n<div class="mb-32 grid text-center lg:mb-0 lg:grid-cols-4 lg:text-left">\\n  <a href="https://nextjs.org/docs?utm_source=create-next-app&utm_medium=default-template-tw&utm_campaign=create-next-app"\\n    class="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"\\n    target="_blank" rel="noopener noreferrer">\\n    <h2 class="mb-3 text-2xl font-semibold">\\n      Docs <span\\n        class="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">-&gt;</span>\\n    </h2>\\n    <p class="m-0 max-w-[30ch] text-sm opacity-50">Find in-depth information about Next.js\\n      features and API.</p>\\n  </a>\\n\\n  <a href="https://nextjs.org/learn?utm_source=create-next-app&utm_medium=default-template-tw&utm_campaign=create-next-app"\\n    class="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"\\n    target="_blank" rel="noopener noreferrer">\\n    <h2 class="mb-3 text-2xl font-semibold">\\n      Learn <span\\n        class="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">-&gt;</span>\\n    </h2>\\n    <p class="m-0 max-w-[30ch] text-sm opacity-50">Learn about Next.js in an interactive\\n      course with&nbsp;quizzes!</p>\\n  </a>\\n\\n  <a href="https://vercel.com/templates?framework=next.js&utm_source=create-next-app&utm_medium=default-template-tw&utm_campaign=create-next-app"\\n    class="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"\\n    target="_blank" rel="noopener noreferrer">\\n    <h2 class="mb-3 text-2xl font-semibold">\\n      Templates <span\\n        class="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">-&gt;</span>\\n    </h2>\\n    <p class="m-0 max-w-[30ch] text-sm opacity-50">Discover and deploy boilerplate example\\n      Next.js&nbsp;projects.</p>\\n  </a>\\n\\n  <a href="https://vercel.com/new?utm_source=create-next-app&utm_medium=default-template-tw&utm_campaign=create-next-app"\\n    class="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"\\n    target="_blank" rel="noopener noreferrer">\\n    <h2 class="mb-3 text-2xl font-semibold">\\n      Deploy <span\\n        class="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">-&gt;</span>\\n    </h2>\\n    <p class="m-0 max-w-[30ch] text-sm opacity-50">Instantly deploy your Next.js site to a\\n      shareable URL with Vercel.</p>\\n  </a>\\n</div>\\n</main>\\n'})();",
  "main.css": "*, ::before, ::after {
    --tw-border-spacing-x: 0;
    --tw-border-spacing-y: 0;
    --tw-translate-x: 0;
    --tw-translate-y: 0;
    --tw-rotate: 0;
    --tw-skew-x: 0;
    --tw-skew-y: 0;
    --tw-scale-x: 1;
    --tw-scale-y: 1;
    --tw-pan-x:  ;
    --tw-pan-y:  ;
    --tw-pinch-zoom:  ;
    --tw-scroll-snap-strictness: proximity;
    --tw-gradient-from-position:  ;
    --tw-gradient-via-position:  ;
    --tw-gradient-to-position:  ;
    --tw-ordinal:  ;
    --tw-slashed-zero:  ;
    --tw-numeric-figure:  ;
    --tw-numeric-spacing:  ;
    --tw-numeric-fraction:  ;
    --tw-ring-inset:  ;
    --tw-ring-offset-width: 0px;
    --tw-ring-offset-color: #fff;
    --tw-ring-color: rgb(59 130 246 / 0.5);
    --tw-ring-offset-shadow: 0 0 #0000;
    --tw-ring-shadow: 0 0 #0000;
    --tw-shadow: 0 0 #0000;
    --tw-shadow-colored: 0 0 #0000;
    --tw-blur:  ;
    --tw-brightness:  ;
    --tw-contrast:  ;
    --tw-grayscale:  ;
    --tw-hue-rotate:  ;
    --tw-invert:  ;
    --tw-saturate:  ;
    --tw-sepia:  ;
    --tw-drop-shadow:  ;
    --tw-backdrop-blur:  ;
    --tw-backdrop-brightness:  ;
    --tw-backdrop-contrast:  ;
    --tw-backdrop-grayscale:  ;
    --tw-backdrop-hue-rotate:  ;
    --tw-backdrop-invert:  ;
    --tw-backdrop-opacity:  ;
    --tw-backdrop-saturate:  ;
    --tw-backdrop-sepia:  ;
    --tw-contain-size:  ;
    --tw-contain-layout:  ;
    --tw-contain-paint:  ;
    --tw-contain-style:  
}
::backdrop {
    --tw-border-spacing-x: 0;
    --tw-border-spacing-y: 0;
    --tw-translate-x: 0;
    --tw-translate-y: 0;
    --tw-rotate: 0;
    --tw-skew-x: 0;
    --tw-skew-y: 0;
    --tw-scale-x: 1;
    --tw-scale-y: 1;
    --tw-pan-x:  ;
    --tw-pan-y:  ;
    --tw-pinch-zoom:  ;
    --tw-scroll-snap-strictness: proximity;
    --tw-gradient-from-position:  ;
    --tw-gradient-via-position:  ;
    --tw-gradient-to-position:  ;
    --tw-ordinal:  ;
    --tw-slashed-zero:  ;
    --tw-numeric-figure:  ;
    --tw-numeric-spacing:  ;
    --tw-numeric-fraction:  ;
    --tw-ring-inset:  ;
    --tw-ring-offset-width: 0px;
    --tw-ring-offset-color: #fff;
    --tw-ring-color: rgb(59 130 246 / 0.5);
    --tw-ring-offset-shadow: 0 0 #0000;
    --tw-ring-shadow: 0 0 #0000;
    --tw-shadow: 0 0 #0000;
    --tw-shadow-colored: 0 0 #0000;
    --tw-blur:  ;
    --tw-brightness:  ;
    --tw-contrast:  ;
    --tw-grayscale:  ;
    --tw-hue-rotate:  ;
    --tw-invert:  ;
    --tw-saturate:  ;
    --tw-sepia:  ;
    --tw-drop-shadow:  ;
    --tw-backdrop-blur:  ;
    --tw-backdrop-brightness:  ;
    --tw-backdrop-contrast:  ;
    --tw-backdrop-grayscale:  ;
    --tw-backdrop-hue-rotate:  ;
    --tw-backdrop-invert:  ;
    --tw-backdrop-opacity:  ;
    --tw-backdrop-saturate:  ;
    --tw-backdrop-sepia:  ;
    --tw-contain-size:  ;
    --tw-contain-layout:  ;
    --tw-contain-paint:  ;
    --tw-contain-style:  
}
.pointer-events-none {
    pointer-events: none
}
.fixed {
    position: fixed
}
.relative {
    position: relative
}
.bottom-0 {
    bottom: 0px
}
.left-0 {
    left: 0px
}
.top-0 {
    top: 0px
}
.z-10 {
    z-index: 10
}
.m-0 {
    margin: 0px
}
.mb-3 {
    margin-bottom: 0.75rem
}
.mb-32 {
    margin-bottom: 8rem
}
.inline-block {
    display: inline-block
}
.flex {
    display: flex
}
.grid {
    display: grid
}
.h-48 {
    height: 12rem
}
.min-h-screen {
    min-height: 100vh
}
.w-full {
    width: 100%
}
.max-w-5xl {
    max-width: 64rem
}
.max-w-\\[30ch\\] {
    max-width: 30ch
}
.flex-col {
    flex-direction: column
}
.place-items-center {
    place-items: center
}
.items-end {
    align-items: flex-end
}
.items-center {
    align-items: center
}
.justify-center {
    justify-content: center
}
.justify-between {
    justify-content: space-between
}
.gap-2 {
    gap: 0.5rem
}
.rounded-lg {
    border-radius: 0.5rem
}
.border {
    border-width: 1px
}
.border-b {
    border-bottom-width: 1px
}
.border-gray-300 {
    --tw-border-opacity: 1;
    border-color: rgb(209 213 219 / var(--tw-border-opacity, 1))
}
.border-transparent {
    border-color: transparent
}
.bg-\\[\\#123456\\] {
    --tw-bg-opacity: 1;
    background-color: rgb(18 52 86 / var(--tw-bg-opacity, 1))
}
.bg-gradient-to-b {
    background-image: linear-gradient(to bottom, var(--tw-gradient-stops))
}
.bg-gradient-to-t {
    background-image: linear-gradient(to top, var(--tw-gradient-stops))
}
.from-white {
    --tw-gradient-from: #fff var(--tw-gradient-from-position);
    --tw-gradient-to: rgb(255 255 255 / 0) var(--tw-gradient-to-position);
    --tw-gradient-stops: var(--tw-gradient-from), var(--tw-gradient-to)
}
.from-zinc-200 {
    --tw-gradient-from: #e4e4e7 var(--tw-gradient-from-position);
    --tw-gradient-to: rgb(228 228 231 / 0) var(--tw-gradient-to-position);
    --tw-gradient-stops: var(--tw-gradient-from), var(--tw-gradient-to)
}
.via-white {
    --tw-gradient-to: rgb(255 255 255 / 0)  var(--tw-gradient-to-position);
    --tw-gradient-stops: var(--tw-gradient-from), #fff var(--tw-gradient-via-position), var(--tw-gradient-to)
}
.p-24 {
    padding: 6rem
}
.p-8 {
    padding: 2rem
}
.px-5 {
    padding-left: 1.25rem;
    padding-right: 1.25rem
}
.py-4 {
    padding-top: 1rem;
    padding-bottom: 1rem
}
.pb-6 {
    padding-bottom: 1.5rem
}
.pt-8 {
    padding-top: 2rem
}
.text-center {
    text-align: center
}
.font-mono {
    font-family: ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace
}
.text-2xl {
    font-size: 1.5rem;
    line-height: 2rem
}
.text-sm {
    font-size: 0.875rem;
    line-height: 1.25rem
}
.font-bold {
    font-weight: 700
}
.font-semibold {
    font-weight: 600
}
.opacity-50 {
    opacity: 0.5
}
.backdrop-blur-2xl {
    --tw-backdrop-blur: blur(40px);
    -webkit-backdrop-filter: var(--tw-backdrop-blur) var(--tw-backdrop-brightness) var(--tw-backdrop-contrast) var(--tw-backdrop-grayscale) var(--tw-backdrop-hue-rotate) var(--tw-backdrop-invert) var(--tw-backdrop-opacity) var(--tw-backdrop-saturate) var(--tw-backdrop-sepia);
    backdrop-filter: var(--tw-backdrop-blur) var(--tw-backdrop-brightness) var(--tw-backdrop-contrast) var(--tw-backdrop-grayscale) var(--tw-backdrop-hue-rotate) var(--tw-backdrop-invert) var(--tw-backdrop-opacity) var(--tw-backdrop-saturate) var(--tw-backdrop-sepia)
}
.transition-colors {
    transition-property: color, background-color, border-color, text-decoration-color, fill, stroke;
    transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
    transition-duration: 150ms
}
.transition-transform {
    transition-property: transform;
    transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
    transition-duration: 150ms
}
.before\\:absolute::before {
    content: var(--tw-content);
    position: absolute
}
.before\\:h-\\[300px\\]::before {
    content: var(--tw-content);
    height: 300px
}
.before\\:w-\\[480px\\]::before {
    content: var(--tw-content);
    width: 480px
}
.before\\:-translate-x-1\\/2::before {
    content: var(--tw-content);
    --tw-translate-x: -50%;
    transform: translate(var(--tw-translate-x), var(--tw-translate-y)) rotate(var(--tw-rotate)) skewX(var(--tw-skew-x)) skewY(var(--tw-skew-y)) scaleX(var(--tw-scale-x)) scaleY(var(--tw-scale-y))
}
.before\\:rounded-full::before {
    content: var(--tw-content);
    border-radius: 9999px
}
.before\\:from-white::before {
    content: var(--tw-content);
    --tw-gradient-from: #fff var(--tw-gradient-from-position);
    --tw-gradient-to: rgb(255 255 255 / 0) var(--tw-gradient-to-position);
    --tw-gradient-stops: var(--tw-gradient-from), var(--tw-gradient-to)
}
.before\\:to-transparent::before {
    content: var(--tw-content);
    --tw-gradient-to: transparent var(--tw-gradient-to-position)
}
.before\\:blur-2xl::before {
    content: var(--tw-content);
    --tw-blur: blur(40px);
    filter: var(--tw-blur) var(--tw-brightness) var(--tw-contrast) var(--tw-grayscale) var(--tw-hue-rotate) var(--tw-invert) var(--tw-saturate) var(--tw-sepia) var(--tw-drop-shadow)
}
.before\\:content-\\[\\'\\'\\]::before {
    --tw-content: '';
    content: var(--tw-content)
}
.after\\:absolute::after {
    content: var(--tw-content);
    position: absolute
}
.after\\:-z-20::after {
    content: var(--tw-content);
    z-index: -20
}
.after\\:h-\\[180px\\]::after {
    content: var(--tw-content);
    height: 180px
}
.after\\:w-\\[240px\\]::after {
    content: var(--tw-content);
    width: 240px
}
.after\\:translate-x-1\\/3::after {
    content: var(--tw-content);
    --tw-translate-x: 33.333333%;
    transform: translate(var(--tw-translate-x), var(--tw-translate-y)) rotate(var(--tw-rotate)) skewX(var(--tw-skew-x)) skewY(var(--tw-skew-y)) scaleX(var(--tw-scale-x)) scaleY(var(--tw-scale-y))
}
.after\\:from-sky-200::after {
    content: var(--tw-content);
    --tw-gradient-from: #bae6fd var(--tw-gradient-from-position);
    --tw-gradient-to: rgb(186 230 253 / 0) var(--tw-gradient-to-position);
    --tw-gradient-stops: var(--tw-gradient-from), var(--tw-gradient-to)
}
.after\\:via-blue-200::after {
    content: var(--tw-content);
    --tw-gradient-to: rgb(191 219 254 / 0)  var(--tw-gradient-to-position);
    --tw-gradient-stops: var(--tw-gradient-from), #bfdbfe var(--tw-gradient-via-position), var(--tw-gradient-to)
}
.after\\:blur-2xl::after {
    content: var(--tw-content);
    --tw-blur: blur(40px);
    filter: var(--tw-blur) var(--tw-brightness) var(--tw-contrast) var(--tw-grayscale) var(--tw-hue-rotate) var(--tw-invert) var(--tw-saturate) var(--tw-sepia) var(--tw-drop-shadow)
}
.after\\:content-\\[\\'\\'\\]::after {
    --tw-content: '';
    content: var(--tw-content)
}
.hover\\:border-gray-300:hover {
    --tw-border-opacity: 1;
    border-color: rgb(209 213 219 / var(--tw-border-opacity, 1))
}
.hover\\:bg-gray-100:hover {
    --tw-bg-opacity: 1;
    background-color: rgb(243 244 246 / var(--tw-bg-opacity, 1))
}
.group:hover .group-hover\\:translate-x-1 {
    --tw-translate-x: 0.25rem;
    transform: translate(var(--tw-translate-x), var(--tw-translate-y)) rotate(var(--tw-rotate)) skewX(var(--tw-skew-x)) skewY(var(--tw-skew-y)) scaleX(var(--tw-scale-x)) scaleY(var(--tw-scale-y))
}
@media (prefers-reduced-motion: reduce) {
    .motion-reduce\\:transform-none {
        transform: none
    }
}
@media (min-width: 1024px) {
    .lg\\:pointer-events-auto {
        pointer-events: auto
    }
    .lg\\:static {
        position: static
    }
    .lg\\:mb-0 {
        margin-bottom: 0px
    }
    .lg\\:flex {
        display: flex
    }
    .lg\\:h-auto {
        height: auto
    }
    .lg\\:w-auto {
        width: auto
    }
    .lg\\:grid-cols-4 {
        grid-template-columns: repeat(4, minmax(0, 1fr))
    }
    .lg\\:rounded-xl {
        border-radius: 0.75rem
    }
    .lg\\:border {
        border-width: 1px
    }
    .lg\\:bg-gray-200 {
        --tw-bg-opacity: 1;
        background-color: rgb(229 231 235 / var(--tw-bg-opacity, 1))
    }
    .lg\\:bg-none {
        background-image: none
    }
    .lg\\:p-0 {
        padding: 0px
    }
    .lg\\:p-4 {
        padding: 1rem
    }
    .lg\\:text-left {
        text-align: left
    }
    .before\\:lg\\:h-\\[360px\\]::before {
        content: var(--tw-content);
        height: 360px
    }
}
@media (prefers-color-scheme: dark) {
    .dark\\:border-neutral-800 {
        --tw-border-opacity: 1;
        border-color: rgb(38 38 38 / var(--tw-border-opacity, 1))
    }
    .dark\\:bg-zinc-800\\/30 {
        background-color: rgb(39 39 42 / 0.3)
    }
    .dark\\:from-black {
        --tw-gradient-from: #000 var(--tw-gradient-from-position);
        --tw-gradient-to: rgb(0 0 0 / 0) var(--tw-gradient-to-position);
        --tw-gradient-stops: var(--tw-gradient-from), var(--tw-gradient-to)
    }
    .dark\\:from-inherit {
        --tw-gradient-from: inherit var(--tw-gradient-from-position);
        --tw-gradient-to: rgb(255 255 255 / 0) var(--tw-gradient-to-position);
        --tw-gradient-stops: var(--tw-gradient-from), var(--tw-gradient-to)
    }
    .dark\\:via-black {
        --tw-gradient-to: rgb(0 0 0 / 0)  var(--tw-gradient-to-position);
        --tw-gradient-stops: var(--tw-gradient-from), #000 var(--tw-gradient-via-position), var(--tw-gradient-to)
    }
    .dark\\:drop-shadow-\\[0_0_0\\.3rem_\\#ffffff70\\] {
        --tw-drop-shadow: drop-shadow(0 0 0.3rem #ffffff70);
        filter: var(--tw-blur) var(--tw-brightness) var(--tw-contrast) var(--tw-grayscale) var(--tw-hue-rotate) var(--tw-invert) var(--tw-saturate) var(--tw-sepia) var(--tw-drop-shadow)
    }
    .dark\\:invert {
        --tw-invert: invert(100%);
        filter: var(--tw-blur) var(--tw-brightness) var(--tw-contrast) var(--tw-grayscale) var(--tw-hue-rotate) var(--tw-invert) var(--tw-saturate) var(--tw-sepia) var(--tw-drop-shadow)
    }
    .before\\:dark\\:bg-gradient-to-br::before {
        content: var(--tw-content);
        background-image: linear-gradient(to bottom right, var(--tw-gradient-stops))
    }
    .before\\:dark\\:from-transparent::before {
        content: var(--tw-content);
        --tw-gradient-from: transparent var(--tw-gradient-from-position);
        --tw-gradient-to: rgb(0 0 0 / 0) var(--tw-gradient-to-position);
        --tw-gradient-stops: var(--tw-gradient-from), var(--tw-gradient-to)
    }
    .before\\:dark\\:to-blue-700\\/10::before {
        content: var(--tw-content);
        --tw-gradient-to: rgb(29 78 216 / 0.1) var(--tw-gradient-to-position)
    }
    .after\\:dark\\:from-sky-900::after {
        content: var(--tw-content);
        --tw-gradient-from: #0c4a6e var(--tw-gradient-from-position);
        --tw-gradient-to: rgb(12 74 110 / 0) var(--tw-gradient-to-position);
        --tw-gradient-stops: var(--tw-gradient-from), var(--tw-gradient-to)
    }
    .after\\:dark\\:via-\\[\\#0141ff\\]\\/40::after {
        content: var(--tw-content);
        --tw-gradient-to: rgb(1 65 255 / 0)  var(--tw-gradient-to-position);
        --tw-gradient-stops: var(--tw-gradient-from), rgb(1 65 255 / 0.4) var(--tw-gradient-via-position), var(--tw-gradient-to)
    }
    .hover\\:dark\\:border-neutral-700:hover {
        --tw-border-opacity: 1;
        border-color: rgb(64 64 64 / var(--tw-border-opacity, 1))
    }
    .hover\\:dark\\:bg-neutral-800\\/30:hover {
        background-color: rgb(38 38 38 / 0.3)
    }
}
@media (min-width: 1024px) {
    @media (prefers-color-scheme: dark) {
        .lg\\:dark\\:bg-zinc-800\\/30 {
            background-color: rgb(39 39 42 / 0.3)
        }
    }
}

",
}
`;

exports[`webpack build > common > errors 1`] = `[]`;

exports[`webpack build > common > warnings 1`] = `[]`;
